changelog/3394.doc.rst

```diff
@@ -0,0 +1,3 @@
+
+- Bring the `conda` command to activate an environment up to date
```

Comment:
```suggestion
```

Reply:
Forgot to mention this should be deleted.

Reply:
ah, no empty lines - makes sense
sorry to cause you so much work